Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob/ReadBlob.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.ComponentModel;
+using System.Threading;
+using System.Text;
+using Azure.Storage.Blobs;
+using Azure;
+#pragma warning disable 1591
+namespace Frends.AzureBlobStorage.ReadBlob
+{
+    public class AzureBlobStorage
+    {
+        /// <summary>
+        /// Read a single file to Azure blob storage using connection string or SAS Token.
+        /// [Documentation](https://github.com/FrendsPlatform/Frends.AzureBlobStorage/tree/main/Frends.AzureBlobStorage.ReadBlob)
```

Comment:
Documentation link should point to Tasks Portal. Please refer to guidelines: https://github.com/FrendsPlatform/FrendsTasks/wiki#task-checklist